src/utils/postParser.js

```diff
@@ -24,6 +25,9 @@ export const parsePost = (post, currentUserName, isPromoted, isList = false) =>
   if (!post) {
     return null;
   }
+
+  post.local_id = generateUuid({ short: true });
```

Comment:
Instead of creating uuid, we can utilize author + permlink as key for ensured uniqueness of each post.

Reply:
discarded generateUUid in favour of author permlink mix